.github/workflows/github-actions.yml

```diff
@@ -5,17 +5,10 @@ concurrency:
   cancel-in-progress: true
 
 on:
-  # run on pushes to certain branches (or PRs targeting those branches)
+  # run on pushes to certain branches
   push:
     branches:
-      - "main"
       - "pull-request/[0-9]+"
-      - "release/v[0-9][0-9].[0-9][0-9].[0-9][0-9]"
-    # run on pushes of new release tags
-    tags:
-      - v[0-9][0-9].[0-9][0-9].[0-9][0-9]
-  # run by clicking buttons in the GitHub Actions UI
-  workflow_dispatch:
```

Comment:
With these removals, this workflow will now only run on `pull-request/` branches, and therefore only on PRs. The new file `build.yaml` (matching naming conventions from the rest of RAPIDS) will be responsible for running on merges to `main` or pushes of new tags.